src/cpp/shared/game/game.h

```diff
@@ -221,6 +240,30 @@ extern int gbInCampaign;
 
 extern int gbGameOver;
 
+extern __int16 giUltimateArtifactBaseX;
+extern __int16 giUltimateArtifactBaseY;
+extern __int16 giUltimateArtifactRadius;
+
+extern char gpcBeforeGSpellLimits[];
+extern int giPlayerInitialResourcesHuman[NUM_DIFFICULTIES][NUM_RESOURCES];
+extern int giPlayerInitialResourcesAI[NUM_DIFFICULTIES][NUM_RESOURCES];
+
+extern int __fastcall FindLastToken(const char *a1, char a2);
+extern char __fastcall StrEqNoCase(int *a1, int *a2);
+
+extern int gbInNewGameSetup;
+extern int iLastMsgNumHumanPlayers;
+extern bool gbSetupGamePosToRealGamePos[];
+extern char gcColorToSetupPos[];
+extern char giVisRange[];
+extern char xIsPlayingExpansionCampaign;
+extern int giCurTurn;
+
+extern __int16 creatureTypesAndQuantities[NUM_FACTIONS][3][3];
```

Comment:
This needs  a more descriptive name.

Reply:
I'm assuming that you are referring to creatureTypesAndQuantities. If so, I'll think of something better. If you have suggestions or preferences for what you want in the name, let me know.